lib/dhcp_wire.ml

```diff
@@ -651,10 +651,10 @@ let options_of_buf buf buf_len =
             (get_ip_tuple_list ())
       in
       let get_string () =  if len < 1 then invalid_arg bad_len else
-          Cstruct.to_string body ~off:0 ~len
+          Cstruct.to_string body
```

Comment:
This is incorrect because len is not (Cstruct.length foo), it's a single byte length encoded within the dhcp option.